docs/framework/docker/aspnetmvc.md

```diff
@@ -50,33 +45,31 @@ The finished application is located in the
 
 ## Prerequisites
 
-At a minimum, your development machine must be running
+The development machine must be running
```

Comment:
Added (or higher)
 I'm thinking by the time new OSs are out this doc will be rewritten several times. I prefer the original  - esp given the very next line is so version specific.